stream-chat-android-client/build.gradle

```diff
@@ -70,12 +70,13 @@ dependencies {
     implementation Dependencies.constraintLayout
     implementation Dependencies.coroutinesCore
     implementation Dependencies.coroutinesAndroid
-    implementation Dependencies.firebaseMessaging
+    api Dependencies.firebaseMessaging
```

Comment:
Why do we need to change it?

Reply:
The `FirebasePushDeviceGenerator` receive by constructor the `FirebaseMessaging` to be used (To follow the previous logic we had to allow our customer to use multiple Firebase Instance if they want). Then, becauses it is exposed by the constructor, it is needed outside of our SDK, and we need to provide it as `api`